nrf_dm/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,51 @@ Changelog
 
 All the notable changes to this project are documented in this file.
 
+main
+****
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added experimental support for the nRF5340 network core:
+
+  * ``lib/nRF5340_CPUNET/soft_float/libnrf_dm.a``
+
+* Added experimental support for a reduced library for the nRF5340 application core:
+
+  * ``lib/nRF5340_CPUAPP/hard_float/libnrf_dm_calc.a``
+
+* Added experimental support for nRF52832:
+
+  * ``lib/nRF52832/hard_float/libnrf_dm.a``
+
+Changes
+=======
+
+* Added a quality indicator per tone.
+* Reduced the memory usage in dm_calc.
+* Changed internal timings which makes this release not compatible with previous releases.
+* Changed the handling of the reports.
+  :c:func:`nrf_dm_calc` and :c:func:`nrf_dm_high_precision_calc` now take the report as an input.
```

Comment:
```suggestion
* Changed the handling of the reports.

  :c:func:`nrf_dm_calc` and :c:func:`nrf_dm_high_precision_calc` now take the report as an input.
```
This separates the two later sentences into their own paragraph under the same bullet. It makes it easier to see them as expanding on the first sentence.